src/run-utils.ts

```diff
@@ -0,0 +1,72 @@
+import fs = require('fs')
+import path = require('path')
+import mod = require('module')
+
+// @ts-ignore
+import execa = require('execa')
+
+// @ts-ignore
+import allModulePaths = require('all-module-paths')
+
+export type Options = { paths?: string[] } & {
+  tag?: string,
+  run?: typeof run,
+  npmClient?: string,
+  prune?: boolean,
+  verbose?: boolean,
+}
+
+export const defaultOptions = {
+  tag: 'latest',
+  run: run,
+  npmClient: 'npm',
+  prune: false,
+  verbose: false,
+}
+
+// export interface defaultOptions {
+//   tag?: string | 'latest',
+//   run?: typeof run | run,
+//   npmClient?: string | 'npm',
+//   prune?: boolean | false,
+//   verbose?: boolean | false,
+// }
+// export type Options = typeof defaultOptions & { paths?: string[] }
```

Comment:
these comments should be removed, I guess

Reply:
oops, yea